File: rps.py
from random import randint


print('Rock Paper Scissors')

o = ['Win!', 'Loss!', 'Tie!', ' > ', ' < ', ' = ']

rps = ['rock', 'paper', 'scissors']

# 0 = rock, 1 = paper, 2 = scissors

a = int(input('(1) rock, (2) paper, (3) scissors: ')) - 1

b = randint(0, 2)

r = (a - b + 2) % 3

print(rps[a] + o[r + 3] + rps[b] + ". " + o[r])

File: rps.cs
using System;



class RPS {


	static void Main() {

		Console.WriteLine("Rock Paper Scissors");


		string[] o = {"Win!", "Loss!", "Tie!", " > ", " < ", " = "},
			 rps = {"rock", "paper", "scissors"};


		Console.Write("(1) rock, (2) paper, (3) scissors: ");

		Random rand = new Random();

		// 0 = rock, 1 = paper, 2 = scissors

		int a = Console.Read() - 49, // convert ASCII representation to integer
		    b = rand.Next(0, 3),
		    r = (a - b + 2) % 3;

		Console.WriteLine(rps[a] + o[r + 3] + rps[b] + ". " + o[r]);

	}

}